src/SingleProject/Resizetizer/src/ResizeImageInfo.cs

```diff
@@ -61,11 +63,15 @@ public static List<ResizeImageInfo> Parse(IEnumerable<ITaskItem> images)
 			foreach (var image in images)
 			{
 				var info = new ResizeImageInfo();
-
+								
 				var fileInfo = new FileInfo(image.GetMetadata("FullPath"));
+
 				if (!fileInfo.Exists)
 					throw new FileNotFoundException("Unable to find background file: " + fileInfo.FullName, fileInfo.FullName);
 
+				if (!ValidImageExtensions.Contains(fileInfo.Extension.ToUpperInvariant()))
+					throw new FileNotFoundException("The file is not a valid image format: " + fileInfo.FullName, fileInfo.FullName);
```

Comment:
Is FileNotFound really the best exception?
File is found, it more of a format exception, isn't it?